examples/ima.amp.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html ⚡>
+<head>
+    <meta charset="utf-8">
+    <title>IMA examples</title>
+    <link rel="canonical" href="amps.html" >
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+    <link href="//vjs.zencdn.net/5.3/video-js.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="http://gvabox.com/sbusolits/amp/css/videojs.ads.css" />
```

Comment:
We're not loading an iframe, though, we're building it on the fly, like facebook. I've tried loading the CSS using prefetch but that doesn't seem to be working. I'm not 100% sure what the issue is, though, because I'm still getting JS errors that I can't seem to debug :/ I'm loading JS scripts that set global variables, and they're loading fine, but the global vars aren't being set.